### DISCUSSION TITLE
Different profiles to connect different own servers or fast switch to option without servers.

### DISCUSSION THREAD

**Discussion by Trogvars** [rustdesk/rustdesk#4525] [Ideas]
I'd like to offer a usefull feature to my mind - profiles for own servers.
For example I have several own servers for different purposes.
Option to switch between them is a solution.
Not to clean and rewrite settings each time but simply switch existing profiles.
Also the option with "No servers" have to be added too.
Attached a screen with menu how it can be done.
<img width="300" alt="rustk1" src="https://github.com/rustdesk/rustdesk/assets/51442309/9848f606-03a7-4258-aef9-08eb6717f210">

**Comment by rustdesk** [comment]
please do not open new thread

**Comment by rustdesk** [comment]
#1258

**Reply by Trogvars** [reply]
Thanks! Didn't find it.
